src/components/Results.tsx

```diff
@@ -37,6 +43,42 @@ const StyledContainer = styled(Box)(({ theme }) => ({
 }));
 
 export const Results: React.FC = () => {
+  const theme = useTheme();
+  const showResultsNumber = useMediaQuery(theme.breakpoints.down('md'));
+
+  const [view, setView] = useState({});
+  const [mode, setMode] = useState<ResultsMode>(ResultsMode.map);
+  const [preventMapView, setPreventMapView] = useState(true);
```

Comment:
what happens if the `preventMapView` is set to `true` in terms of UI?

Reply:
prevent list auto close on scroll